ui/src/components/ui/device-control-panel/tabs/device-buttons-control/button-control.tsx

```diff
@@ -0,0 +1,42 @@
+import { cloneElement } from 'react'
+import cn from 'classnames'
+import { Button, Tooltip } from '@vkontakte/vkui'
+
+import styles from './device-buttons-control.module.css'
+
+import type { ReactElement } from 'react'
+
+type ButtonControlProps = {
+  icon?: ReactElement
+  iconWidth?: number
+  iconHeight?: number
+  tooltipText: string
+  children?: string
+  className?: string
+  appearance?: 'accent' | 'neutral' | 'positive' | 'negative' | 'overlay' | 'accent-invariable'
+  onClick?: () => void
+}
+
+export const ButtonControl = ({
```

Comment:
I think at one point we should document every component we have. Or make a stylebook. Not that important, but will be nice to have when more people will want to join the project